src/main/java/ch/jalu/configme/configurationdata/CommentsConfiguration.java

```diff
@@ -59,4 +61,24 @@ public void setComment(@NotNull String path, @NotNull String @NotNull ... commen
     public @NotNull @UnmodifiableView Map<String, @UnmodifiableView List<String>> getAllComments() {
         return Collections.unmodifiableMap(comments);
     }
+
+    /**
+     * Adds a footer comment so they are stored in the same map under a reserved key.
+     * This allows footer comments to be written at the end of the configuration file.
```

Comment:
I think callers don't really need to know about the implementation details, probably something like this would be enough?
```java
Adds the given lines as footer comments. They will be written at the end of the configuration file.
```